internal/service/user-service.go

```diff
@@ -37,7 +37,7 @@ func (s *UserService) Save(user dto.UserSaveRequestDTO) (dto.UserResponseDTO, er
 
 // GetAll is used to get all available in database users' information.
 // Returns all available UserResponseDTO.
-func (s *UserService) GetAll() ([]dto.UserResponseDTO, error) {
+func (s *UserServiceImpl) GetAll() ([]dto.UserResponseDTO, error) {
```

Comment:
Is it a repository?

Reply:
I'll check repository pattern and probably change this.